src/org/traccar/model/Position.java

```diff
@@ -54,6 +54,7 @@
     public static final String KEY_APPROXIMATE = "approximate";
     public static final String KEY_THROTTLE = "throttle";
     public static final String KEY_MOTION = "motion";
+    public static final String KEY_GUARD = "guard";
```

Comment:
it is from Navis protocol
````
 Статус охраны / режим
тревожного оповещения
U8 Битовое поле:
0…2 Биты
0 – наблюдение (нет охраны);
1 - профиль охраны №1 (полная охрана);
2 - профиль охраны №2;
3…5 Биты Резерв
6 Бит
0 – рабочий режим
1 – тестовый режим
7 Бит
0 – штатная работа
1 – режим тревожного оповещения
````
I translated it to "Guard status" If "armed" is better, then OK.